packages.yml

```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/netsuite_source
-  version: [">=0.5.0", "<0.6.0"]
\ No newline at end of file
+# - package: fivetran/netsuite_source
+#   version: [">=0.5.0", "<0.6.0"]
+# - local: ../Netsuite/dbt_netsuite_source
+- git: https://github.com/fivetran/dbt_netsuite_source.git
+  revision: bugfix/passthrough
+  warn-unpinned: false
```

Comment:
This PR is ready to go, there are just a few final changes needed to be addressed before we release. @fivetran-jamie would you mind addressing the [PR changes](https://github.com/fivetran/dbt_fivetran_utils/pull/81#pullrequestreview-1072856799). Once those are updated we can cut the release and this will be good to go!.